rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -1274,3 +1276,9 @@ fn disable_collision() {
 fn exit_water() {
     AMyCharacter::exit_water();
 }
+#[rebo::function("Tas::open_maps_folder")]
+fn open_maps_folder() {
+    if let Err(err) = opener::open(map_path().as_os_str().to_str().unwrap()) {
```

Comment:
`map_path()` returns a `PathBuf`, which derefs to `&Path`, which implements `AsRef<OsStr>`, which `opener::open` expects as argument. No need to convert it to a `&str` which may fail if the filename contains invalid UTF-8 (as filesnames use WTF-8 encodeng).

Reply:
Sweet. Will address that rn.